src/css/_markdown.scss

```
@@ -0,0 +1,151 @@
+}
+
+// External link icon indicator
+.theme-doc-markdown a[target="_blank"]::after {
+  content: "";
+  display: inline-block;
+  width: 0.75em;
+  height: 0.609375em;
+  margin-left: 0.2em;
+  vertical-align: baseline;
+  transform: rotate(-45deg);
+  background-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)